### PR TITLE
Add doPast for Korean

### PR DESCRIPTION
**Pull request by minsooshin**

### CODE REVIEW

File: test/korean_tests.js [hunk 1]
```diff
@@ -125,67 +139,70 @@ describe('Korean', () => {
...
-   });
-   it('should conjugate verbs with stem ending with ㅂ consonant correctly',
-   () => {
-     const kc = new Korean();
-     let futureWord = kc.conjugate('춥다', {tense: 'future'});
-     expect(futureWord).to.equal('추울 거야');
-
-     futureWord = kc.conjugate('덥다', {tense: 'future'});
-     expect(futureWord).to.equal('더울 거야');
-   });
```
Comment (minsooshin) [inline test/korean_tests.js:160]: I removed this test case because `춥다` and `덥다` are adjectives, not verbs.

File: src/korean.js [hunk 2]
```diff
@@ -129,8 +157,17 @@ class Korean {
   }
 
   doPast (word) {
-    console.info(word);
-    // stuff for past tense
+    const presentTense = this.doPresent(word);
```
Comment (minsooshin) [inline src/korean.js:174]: @thisisrai I am using `doPresent`

Comment (thisisrai) [reply src/korean.js:174]: @minsooshin Fasho, my mistake on getting the doPresent changes confused with doPast.

**Comment by songz** [comment]
@minsooshin sorry for the late response, would you mind fixing MRs? @yjlim5 and I was busy preparing for our wedding this week :)

**Comment by thisisrai** [comment]
That's awesome, congrats!!

**Comment by minsooshin** [comment]
@songz congrats!!! Sure, I will fix the conflicts.

**Comment by minsooshin** [comment]
@songz I am not sure how the Travis is set, but after I commit the fix for conflict, the CI is not started.

File: test/korean_tests.js [hunk 3]
```diff
@@ -101,7 +101,21 @@ describe('Korean', () => {
       expect(presentWord).to.equal('불러');
 
       presentWord = kc.conjugate('서두르다', {tense: 'present'});
-      presentWord = kc.conjugate('서둘러', {tense: 'present'});
+      expect(presentWord).to.equal('서둘러');
```
Comment (minsooshin) [inline test/korean_tests.js:104]: this is fix for testing of 서둘러

**Review by yjlim5 — APPROVED** [review]

**Comment by yjlim5** [comment]
Not sure why Travis CI didn't get triggered, but I ran `npm run eslint` locally and it's clean. I'll merge it in. Thanks @minsooshin and @thisisrai!

**Pull request merged**